### PR TITLE
Add links from issue 677

### PR DESCRIPTION
**Pull request by djwfyi**
I added the accessibility and bias notes to the Style Guides topic.
That one is in markdown, so I was able to preview and looked okay.

Other links I added to the Guide index.
That file is in rst, and I could not get my preview to work, now do I have Sphinx installed. Worth checking it.

### CODE REVIEW

**Comment by djwfyi** [comment]
Addresses #677

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (plaindocs) [inline .vscode/settings.json:1]: This generated file doesn't really belong here. 
Could you delete it, and and add it to `.gitignore`

**Review by plaindocs — COMMENTED** [review]
This is excellent stuff! Thanks @dwhitelf 

It has the same 3 deleted video files that should be removed. Let me know if you'd like a hand with those.

**Comment by dwhitelf** [comment]
Which videos?

File: docs/videos/eu/2015/imposter-no-more-how-tech-writers-can-shed-self-doubt-embrace-uncertainty-and-surf-the-upcoming-swerve-in-technical-documentation-rmacnamara.rst [hunk 2]
```diff
@@ -1,8 +0,0 @@
-IMPOSTER NO MORE: How Tech Writers Can Shed Self-Doubt, Embrace Uncertainty, and Surf the Upcoming Swerve in Technical Documentation
```
Comment (plaindocs) [inline docs/videos/eu/2015/imposter-no-more-how-tech-writers-can-shed-self-doubt-embrace-uncertainty-and-surf-the-upcoming-swerve-in-technical-documentation-rmacnamara.rst:1]: Hi @dwhitelf I'm talking about these 3 files which are unrelated to your PR. 

There is an error in the build system somewhere that deleted the files, and then 
I think you ran `git add -A` and which "added" the deleted files to the commit.

Comment (djwfyi) [reply docs/videos/eu/2015/imposter-no-more-how-tech-writers-can-shed-self-doubt-embrace-uncertainty-and-surf-the-upcoming-swerve-in-technical-documentation-rmacnamara.rst:1]: I don't think I have the git skills to figure out how to get them back.

Comment (plaindocs) [reply docs/videos/eu/2015/imposter-no-more-how-tech-writers-can-shed-self-doubt-embrace-uncertainty-and-surf-the-upcoming-swerve-in-technical-documentation-rmacnamara.rst:1]: No worries. I often revert to basics for this, and just copy the files back and commit them again.

Comment (plaindocs) [reply docs/videos/eu/2015/imposter-no-more-how-tech-writers-can-shed-self-doubt-embrace-uncertainty-and-surf-the-upcoming-swerve-in-technical-documentation-rmacnamara.rst:1]: Sorted! Thanks

**Comment by samanthahamilton** [comment]
Hi, @plaindocs and @dwhitelf,

I love the resource inclusions to address accessibility and bias. One small note--and I'm not sure where it's best to "put" this--there is a spelling error on the page. Under the heading "Thinking about accessibility and bias," the last paragraph reads, "Writing for bias asks you to consider the meanings and **_orgins_** of your word choices and how those might be perceived or understood by your readers." 

Cheers, Sam.

**Comment by djwfyi** [comment]
Thanks @samanthahamilton 
Fixed by #1309
